generate_token.py

```
@@ -0,0 +1,19 @@
+from livekit.api import AccessToken, VideoGrants
+
+# These are the default keys for a local LiveKit server
+API_KEY = "devkey"
+API_SECRET = "secret"
```

Ruff (0.14.6)

Possible hardcoded password assigned to: "API_SECRET"

(S105)